src/window.rs

```diff
@@ -18,6 +18,9 @@ pub use crate::icon::{BadIcon, Icon};
 #[doc(inline)]
 pub use cursor_icon::{CursorIcon, ParseError as CursorIconParseError};
 
+#[doc(inline)]
+pub use raw_window_handle;
```

Comment:
Is there any reason for the `doc(inline)`? I think we should emphasize that it is an external crate.

Reply:
mostly to easily read docs. We could move it to `winit::reexports` similar, to what sctk does.

Reply:
I see, that makes sense. If we start re-exporting a lot of crates we can do that in the future, but for now I'm fine with this.